website/templates/home.mako

```diff
@@ -45,6 +45,39 @@
         window.contextVars = $.extend(true, {}, window.contextVars, {
             dashboardInstitutions: ${ dashboard_institutions | sjson, n},
         });
+        var FLAGS = ${ flags | sjson, n }
```

Comment:
Rather than writing this code here, could we instead request the JS file, as documented here: https://waffle.readthedocs.io/en/latest/usage/javascript.html ?

The only difference is that instead of using `<script src="{% url 'wafflejs' %}"></script>`, we would need to input the url ourselves. We could put the URL in the global mako context, i.e. in `website.routes.get_globals` so that it can be included in a script tag. Something like

```python
# website/routes
def get_globals():
    #...
    return {
         # ...
          'wafflejs_url': reverse('wafflejs')
    }
```


```mako
## base.mako

<script src="${wafflejs_url}"></script>